src/commands/new.ts

```diff
@@ -195,8 +195,12 @@ export default {
     // #endregion
 
     // #region Project Path
+    const parsePath = (p: string) =>
+      p.startsWith("~") ? path(p.replace("~", filesystem.homedir())) : path(p)
```

Comment:
For some reason, `filesystem.path` does not seem to like `~/Code/PizzaApp`. So instead, we are replacing the `~` with the actual home dir path